config/manager/kustomization.yaml

```diff
@@ -13,4 +13,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: quay.io/openshift_sandboxed_containers/openshift-sandboxed-containers-operator
-  newTag: 1.7.0
+  newTag: v1.7.0
```

Comment:
Do we want to keep this change ? It seems we didn't with previous Operator SDK updates.

Reply:
In latest Makefile, the images are referenced with `v${VERSION}`. So I think we'll need this change no?